File: 백준/Bronze/2566. 최댓값/최댓값.js
const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

let maxNumber = -1;
let maxNumberCoordinate = [0, 0];

for (let i = 0; i < 9; i++) {
  const row = input[i].split(" ").map(Number);
  for (let j = 0; j < 9; j++) {
    if (row[j] > maxNumber) {
      maxNumber = row[j];
      maxNumberCoordinate = [i + 1, j + 1];
    }
  }
}

console.log(maxNumber);
console.log(`${maxNumberCoordinate[0]} ${maxNumberCoordinate[1]}`);

File: 백준/Bronze/2566. 최댓값/최댓값.py
matrix = []
max_number = -1
max_number_coordinate = (0, 0)

for i in range(9):
    row = list(map(int, input().split()))
    matrix.append(row)
    for j in range(9):
        if row[j] > max_number:
            max_number = row[j]
            max_number_coordinate = (i + 1, j + 1)

print(max_number)
print(f"{max_number_coordinate[0]} {max_number_coordinate[1]}")